test/types/populate.test.ts

```diff
@@ -228,4 +228,29 @@ async function _11532() {
   if (!leanResult) return;
   expectType<string>(leanResult.child.name);
   expectError(leanResult?.__v);
+}
+
+async function gh11710() {
```

Comment:
Is this accidently added?

Reply:
No, just wrong base. We should merge this into 6.4.